Detectors/TPC/base/include/TPCBase/Defs.h

```diff
@@ -51,26 +51,47 @@ enum class PadSubset : char {
   Partition,  ///< Partitions (up to 36*5)
   Region      ///< Regions (up to 36*10)
 };
-/// Pad centres as 2D float
-typedef Point2D<float> PadCentre;
-typedef Point2D<float> GlobalPosition2D;
-typedef Point2D<float> LocalPosition2D;
-typedef Point3D<float> GlobalPosition3D;
-typedef Point3D<float> LocalPosition3D;
+/// default point definitions
+template <typename T>
+using Point3D     = ROOT::Math::PositionVector3D<ROOT::Math::Cartesian3D<T>, ROOT::Math::DefaultCoordinateSystemTag>;
+template <typename T>
+using Point3Dglob = ROOT::Math::PositionVector3D<ROOT::Math::Cartesian3D<T>, ROOT::Math::GlobalCoordinateSystemTag>;
+template <typename T>
+using Point3Dloc  = ROOT::Math::PositionVector3D<ROOT::Math::Cartesian3D<T>, ROOT::Math::LocalCoordinateSystemTag>;
```

Comment:
If we differentiate between the local and global coordinates on the type level to avoid ambiguities, we still have a problem of having too many "local" frames (sectors, "tracking" frames for ITS etc.). 

Concerning the Point3Dglob and Point3Dloc: this may create a problem for transformations. The GenVector::Transform3D takes as input and transformed point class the same ````PositionVector3D<CoordSystem>````, of we want to be able to transform Point3Dloc to Point3Dglob, we will need to rewrite this class.